### ISSUE TITLE
a11y: Color Contrast - Cluster Management Create

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13810]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF 
Ratio: 3.16:1
Main Content
- Cancel button"

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 174 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #2F68DF
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> - Cancel button (on focus and hover)
>  
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.
> "

Issue 175 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #2F68DF
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> - Cancel button (on focus and hover)
> - Select Existing (on focus and hover)
> 
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF
> Ratio: 2.00:1
> Main Content
> - Your AWS Access Key and Your AWS Secret Key placeholder text 
> 
> Foreground Text Color: #FFFFFF
> Background Text Color: #B6B7C2
> Ratio: 2:1
> Main Content
> -  us-west-2 (Selected option in dropdown)
>  
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.
> "
